Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,58 @@ click: function () { closeClicked.raiseEvent(this); }');
         closeElement.innerHTML = '&times;';
         infoElement.appendChild(closeElement);
 
-        var infoBodyElement = document.createElement('div');
-        infoBodyElement.className = 'cesium-infoBox-body';
-        infoElement.appendChild(infoBodyElement);
-
-        var descriptionElement = document.createElement('div');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('allowfullscreen', true);
+        infoElement.appendChild(frame);
 
         var viewModel = new InfoBoxViewModel();
-        this._viewModel = viewModel;
+        knockout.applyBindings(viewModel, infoElement);
+
+        //We inject default css into the content iframe,
+        //end users can remove it or add their own via the exposed frame property.
+        var cssLink = document.createElement("link");
+        cssLink.href = buildModuleUrl('Widgets/InfoBox/InfoBoxDescription.css');
+        cssLink.rel = "stylesheet";
+        cssLink.type = "text/css";
 
-        knockout.applyBindings(this._viewModel, infoElement);
+        //div to use for description content.
+        var frameContent = document.createElement("div");
+        frameContent.className = 'cesium-infoBox-description';
+
+        this._container = container;
+        this._element = infoElement;
+        this._frame = frame;
+        this._viewModel = viewModel;
+        this._descriptionSubscription = undefined;
+
+        var that = this;
+
+        //We can't actually add anything into the frame until the load event is fired
+        frame.addEventListener('load', function() {
+            var frameDocument = frame.contentDocument;
+            frameDocument.head.appendChild(cssLink);
+            frameDocument.body.appendChild(frameContent);
+
+            //Once the frame is initialized, we can enable sandboxing.
+            frame.setAttribute('sandbox', 'allow-popups allow-forms'); // allow-same-origin allow-pointer-lock allow-scripts allow-top-navigation
```

Comment:
So it turns out, this is ineffective here, no sandboxing is being applied.  @shunter and I were talking over the options, and basically the simplest thing to do is move this back to line 76 where it was, and add back the `allow-same-origin` tag.  Scott was championing a more complex solution involving blobs and postMessage, but I'm concerned that there may be visual glitches involved (due to async resize calculations and/or the old SVG-in-blob bug).  May be worth a try, but I talked Scott into agreeing that just adding back `same-origin` is fine.  This also makes life easier for trusted iframe content, for apps where scripts are intentionally enabled.

Separate from all this, Scott noticed that you're constructing the `div` and the CSS on the parent document, not on the iframe's document.  You should probably move that construction inside the load event, and construct on the iframe's document.